install-jar.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
```

Comment:
Can you make this filename more descriptive and put a comment on top? Or just don't include it in the commit?

Reply:
I will remove this file from the commit. I had tried to add tests to this feature, but it has turned out that it's not that straightforward task. I had to enable ```sun.misc.Unsafe``` access to be able to allocate native memory and Scala (sbt) just makes this impossible (ate least for me). I have zero experience in Scala and its tooling.  In Java 9+ the access to this class is prohibited by default, so you have to specify additional command line args:
`java  --add-opens jdk.unsupported/sun.misc=ALL-UNNAMED --add-opens java.base/jdk.internal.misc=ALL-UNNAMED `

For some reason this does not work with ```sbt```. Probably I did something wrong.

Reply:
Can you try to add these options to https://github.com/luben/zstd-jni/blob/master/build.sbt#L39 . I just pushed a change to run the tests in forked JVM so these options will apply.

Reply:
I recently moved repo fork to a new owner (organization); http://www.github.com/crootdata/zstd-jni. The current PR is in some kind of a zombie state after that. I am going to close this PR and open new one, @luben . What do you think?

Reply:
```install-jar.sh``` has been removed.

Reply:
> I recently moved repo fork to a new owner (organization) - https://github.com/carrotdata/zstd-jni/tree/master. The current PR is in some kind of a zombie state after that. I am going to close this PR and open new one, @luben . What do you think?

Sure, we can continue on a new PR

Reply:
>  I just pushed a change to run the tests in forked JVM so these options will apply.
Hmm, it was not so easy - forking broke builds on Windows and Mac OS, not sure why. So I reverted that